### PR TITLE
feat: add tests for ecommer ecommerce-io

### PR DESCRIPTION
**Pull request by infantesimone**
#### List of Changes

- add collection postman to test happy path for ecommerce for io in uat
- add pipeline to run api test
#### How Has This Been Tested?

https://dev.azure.com/pagopaspa/pagoPA-projects/_build/results?buildId=127509&view=logs&j=408b4e4c-6e94-5189-521b-a5cea81b8ec4&t=2fef5b5f-da1e-542f-df95-15e87f6ef239

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/api-tests/ecommerce-for-io/ecommerce-for-io-uat.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+						"v1",
+						"wallets"
+					]
+				},
+				"description": ""
+			},
+			"response": []
+		},
+		{
+			"name": "Payment Info by rptId",
```
Comment (pietro-tota) [inline src/api-tests/ecommerce-for-io/ecommerce-for-io-uat.postman_collection.json:70]: Add authorization header for `Payment Info by rptId`api call

File: src/api-tests/ecommerce-for-io/ecommerce-for-io-uat.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+						"io",
+						"v1",
+						"sessions"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get eCommerce transaction by TRANSACTION_ID",
```
Comment (pietro-tota) [inline src/api-tests/ecommerce-for-io/ecommerce-for-io-uat.postman_collection.json:250]: Add authorization for `Get eCommerce transaction by TRANSACTION_ID` api call

File: src/api-tests/ecommerce-for-io/ecommerce-for-io-uat.postman_collection.json (hunk 1 above)
Comment (infantesimone) [reply src/api-tests/ecommerce-for-io/ecommerce-for-io-uat.postman_collection.json:70]: added, but I don't know if it's right :-)

Comment (pietro-tota) [reply src/api-tests/ecommerce-for-io/ecommerce-for-io-uat.postman_collection.json:70]: In [openapi](https://editor.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pagopa-infra/b01a960db047fcbb94222216fd149670183b8892/src/domains/ecommerce-app/api/ecommerce-io/v1/_openapi.json.tpl) we have marked as secured all api calls, including verification. Should we remove authorization for verification api call?

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**